pkg/cmd/server/api/types.go

```diff
@@ -235,6 +235,11 @@ type ServingInfo struct {
 	ServerCert CertInfo
 	// ClientCA is the certificate bundle for all the signers that you'll recognize for incoming client certificates
 	ClientCA string
+	// MaxRequestsInFlight is the number of concurrent requests allowed to the server. If zero, no limit.
```

Comment:
this gets used by lots of things (etcd, master, assets, node)... probably shouldn't add options here unless we support them uniformly